curriculum/challenges/english/14-responsive-web-design-22/learn-intermediate-css-by-building-a-cat-painting/646c586be7180e40ddf74ff6.md

```diff
@@ -0,0 +1,47 @@
+---
+id: 646c586be7180e40ddf74ff6
+title: Step 4
+challengeType: 0
+dashedName: step-4
+---
+
+# --description--
+
+To get your cat painting started, give your `body` element a `background-color` of `#c9d2fc`.
+
+# --hints--
+
+Test 1
+
+```js
+
+```
+
+# --seed--
+
+## --seed-contents--
+
+```html
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>fCC Cat Painting</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body></body>
+</html>
+```
+
+```css
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
```

Comment:
@naomi-lgbt 
For the new JS projects, some of the css files do have this minimal css reset here.

But for this particular certification, I do think we should stay consistent with what the other projects are doing.

After a quick glance it seems like most of the other projects are setting the `box-sizing` for the html element.

```css
html {
  box-sizing: border-box;
}
```
@Ksound22 I vote we use that instead

Reply:
```css
html {
  box-sizing: border-box;
}
```

`box-sizing` isn't inherited by default, if there are projects that use the syntax above, it needs to be looked at if they actually need that property, so this issue doesn't repeat itself https://github.com/freeCodeCamp/freeCodeCamp/issues/49284

Reply:
@Sboonny 

After looking more closely at all of the practice projects there doesn't seem to be a consistent use of where the box-sizing is set. 


The balance sheet project and the building a piano project set the box-sizing for the html element.

```
html {
  box-sizing: border-box;
}
```
The photo gallery, nutrition label, and city skyline set the box sizing to the `*` selector. 

```
* {
  box-sizing: border-box;
}
```
If this cat painting project falls after the magazine project, then we can just leave the current implementation as it is since campers have already been taught that. 

Otherwise, we could just apply the box sizing to the `*` selector

Reply:
> Why are we writing this for the campers?

Just a few resets I always put in my CSS. I'll do what Jessica suggested about it.

Reply:
> @Sboonny
> 
> After looking more closely at all of the practice projects there doesn't seem to be an inconsistent use of where the box-sizing is set.
> 
> The balance sheet project and the building a piano project set the box-sizing for the html element.
> 
> ```
> html {
>   box-sizing: border-box;
> }
> ```
> 
> The photo gallery, nutrition label, and city skyline set the box sizing to the `*` selector.
> 
> ```
> * {
>   box-sizing: border-box;
> }
> ```
> 
> If this cat painting project falls after the magazine project, then we can just leave the current implementation as it is since campers have already been taught that.
> 
> Otherwise, we could just apply the box sizing to the `*` selector

Which should I use?
```css
html {
   box-sizing: border-box;
```
or

```css
* {
   box-sizing: border-box;
}
```

Reply:
@Ksound22 

I would just go with using the `*` selector. 

```
* {
   box-sizing: border-box;
}
```
You will also need to create a new step before this one to have campers add it. 

You can use the Insert step option mentioned [here](https://contribute.freecodecamp.org/#/how-to-work-on-practice-projects?id=insert-step) in the docs to do that.